.aicoe-ci.yaml

```diff
@@ -1,2 +1,15 @@
 ---
-check: []
+check:
+  - overlays-build
+overlays_dir: docs/content
```

Comment:
@harshad16 just want to confirm, is this the only way to specify a path to the build path? 

Is there anyway we can have this _per overlay_, so that in the future we can have multiple builds if we need it, pointing the build working directory to different parts of the `apps` repo?

Reply:
Also shouldn't it be: 
```
overlays_dir: docs
```
?

Reply:
Based on our discussion in call. It doesn't seem like we need to use `overlays_dir` here. 

Also, we don't want s2i to copy the whole repo, but to force it to only copy `notebooks` (what we want) we need to be able to specify the `context_path` var, as per @harshad16. It is currently not possible to specify this in the `aicoe-ci.yaml`.

A workaround suggested by @4n4nd is to just update the `source` strategy to `Dockerfile` and write our own dockerfile that copies the contents from the notebooks folder into the image.

Reply:
> @harshad16 just want to confirm, is this the only way to specify a path to the build path?
> 
> Is there anyway we can have this _per overlay_, so that in the future we can have multiple builds if we need it, pointing the build working directory to different parts of the `apps` repo?

Seconding that I don't believe we can do this without extending the AICoE app to support different paths from which to build certain overlays. Also, I believe we discussed that workaround briefly in Face In the Clouds so I know that Harshad is aware of it and said it could work.